QUICKSTART.md Instructions are inaccurate for RHEL-based systems

Hello there

I recently tried to compile the Kernel Modules when a compilation error lead me to re-read the QUICKSTART.md. I noticed that the command "make prepare" is not needed for RHEL-based systems and only leads to confusion as it fails when following the instructions:

```
[root@devsystem ~]# yum install kernel-headers kernel-devel elfutils-libelf-devel
Package kernel-headers-3.10.0-1160.80.1.el7.x86_64 already installed and latest version
Package kernel-devel-3.10.0-1160.80.1.el7.x86_64 already installed and latest version
Package elfutils-libelf-devel-0.176-5.el7.x86_64 already installed and latest version
Nothing to do
[root@devsystem ~]# cd /usr/src/kernels/$(uname -r)
[root@devsystem 3.10.0-1160.80.1.el7.x86_64]# make oldconfig
  HOSTCC  scripts/basic/fixdep
  HOSTCC  scripts/basic/bin2c
  HOSTCC  scripts/kconfig/conf.o
  HOSTCC  scripts/kconfig/zconf.tab.o
  HOSTLD  scripts/kconfig/conf
scripts/kconfig/conf --oldconfig Kconfig
#
# configuration written to .config
#
[root@devsystem 3.10.0-1160.80.1.el7.x86_64]# make prepare
scripts/kconfig/conf --silentoldconfig Kconfig
make[1]: *** No rule to make target `arch/x86/syscalls/syscall_32.tbl', needed by `arch/x86/syscalls/../include/generated/uapi/asm/unistd_32.h'.  Stop.
make: *** [archheaders] Error 2
```

I first thought the compilation was unsuccessful because i have not "prepared" my sources, but unless one builds the Kernel Modules with cloned Kernel source code, this command is not needed. After correcting the error in my setup. I compiled the modules sucessfully with Kernel 3.10, and it was sufficient to install the packages mentioned above.

Let me know if you need further information/clarification about this.

Thank you and kind regards
Matthias

--- a/QUICKSTART.md
+++ b/QUICKSTART.md
@@ -100,14 +100,6 @@ Note: RedHat Linux is not officially supported!
 
     ``# ln -s /usr/src/kernels/$(uname -r) /usr/src/linux``
 
-- Prepare the kernel headers:
-
-  ```
-  # cd /usr/src/linux
-  # make oldconfig
-  # make prepare
-  ```
-
 ### Manjaro / Arch Linux
 
 Note: Manjaro/Arch Linux is not officially supported!

